tests/core/test_circuit_breaker.py

```
@@ -0,0 +1,199 @@
+                await circuit_breaker.execute(fail_func)
+
+        circuit_breaker.reset()
+        assert circuit_breaker.state == CircuitState.CLOSED
+        assert circuit_breaker.stats.failure_count == 0
+        assert circuit_breaker.stats.success_count == 0
+
+class TestCircuitBreakerRegistry:
+    """Test suite for CircuitBreakerRegistry class."""
+
+    @pytest.mark.asyncio
+    async def test_circuit_creation(self, registry):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_circuit_creation`

Add return type annotation: `None`

(ANN201)